src/main/java/de/unistuttgart/overworldbackend/data/World.java

```diff
@@ -0,0 +1,33 @@
+package de.unistuttgart.overworldbackend.data;
+
+import java.util.List;
+import java.util.Set;
+import javax.persistence.*;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.FieldDefaults;
+
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class World extends Area {
+
+  @OneToMany(cascade = CascadeType.ALL)
+  List<Dungeon> dungeons;
```

Comment:
```suggestion
  Set<Dungeon> dungeons;
```
?
Or would we then lose the order?
If yes, we should probably update our data model

Reply:
Because of our oneToMany and manyToOne relation between dungeon and world, a the implementation with a set gets stuck in an StackOverflowError. Because of this we have to use List or remove the relation.